eng/Versions.props

```diff
@@ -47,16 +47,16 @@
 
   -->
   <PropertyGroup Label="Automated">
-    <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>3.1.8-servicing.20413.6</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
-    <MicrosoftAspNetCoreTestingPackageVersion>3.1.8-servicing.20413.6</MicrosoftAspNetCoreTestingPackageVersion>
-    <MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>3.1.8-servicing.20413.6</MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>3.1.3</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftExtensionsHashCodeCombinerSourcesPackageVersion>3.1.8-servicing.20413.6</MicrosoftExtensionsHashCodeCombinerSourcesPackageVersion>
+    <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>3.1.8-servicing.20413.8</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
+    <MicrosoftAspNetCoreTestingPackageVersion>3.1.8-servicing.20413.8</MicrosoftAspNetCoreTestingPackageVersion>
+    <MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>3.1.8-servicing.20413.8</MicrosoftExtensionsCommandLineUtilsSourcesPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>3.1.0</MicrosoftExtensionsDependencyModelPackageVersion>
```

Comment:
@mmitche @wtgodbe I noticed a previous dependency update took 3.1.6 versions back to 3.1.3. Why are we going back to 3.1.0 now❔ Is the strict coherency fix just to add this package and Microsoft.DotNet.PlatformAbstractions to the dependencies in dotnet/extensions❔

Reply:
@dougbu Here is what darc is telling me:

```
PS C:\r\aspnetcore-tooling> darc update-dependencies --strict-coherency --coherency-only
Checking for coherency updates...
Coherency updates failed for the following dependencies:
  Unable to update System.Diagnostics.DiagnosticSource to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup @ fcfdef8d6ba770e4d7ba9efefe2591d8c33a1fc6 does not contain dependency System.Diagnostics.DiagnosticSource
    - Add the dependency to https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.Reflection.Metadata to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup @ fcfdef8d6ba770e4d7ba9efefe2591d8c33a1fc6 does not contain dependency System.Reflection.Metadata
    - Add the dependency to https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.DependencyModel to have coherency with Microsoft.Extensions.Logging: https://github.com/dotnet/extensions @ 216db3e325add5768a2c620c19c8d6a1ae420062 does not contain dependency Microsoft.Extensions.DependencyModel
    - Add the dependency to https://github.com/dotnet/extensions.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
Error: Failed to update coherent parent tied dependencies
```